packages/react-ai/src/components/AIConversation/AIConversation.tsx

```diff
@@ -17,29 +17,20 @@ import { MessageList } from './views/default/MessageList';
 import { Form } from './views/default/Form';
 import { PromptList } from './views/default/PromptList';
 import { ComponentClassName } from '@aws-amplify/ui';
-import {
-  AIConversationProvider,
-  AIConversationProviderProps,
-} from './AIConversationProvider';
+import { AIConversationProvider } from './AIConversationProvider';
 import { useSetUserAgent } from '@aws-amplify/ui-react-core';
 import { VERSION } from '../../version';
 import { DefaultMessageControl } from './views/Controls/DefaultMessageControl';
 
-function Provider({
-  actions,
+interface AIConversationBaseProps
+  extends AIConversationProps,
+    AIConversationInput {}
+
+function AIConversationBase({
   avatars,
   controls,
-  handleSendMessage,
-  messages,
-  responseComponents,
-  suggestedPrompts,
-  variant,
-  isLoading,
-  displayText,
-  allowAttachments,
-  messageRenderer,
-  children,
-}: AIConversationProviderProps): JSX.Element {
+  ...rest
```

Comment:
Moving all these to just rest, we only need the nested configurations like avatars and controls to merge those with the defaults.